package/com.unity.formats.usd/Tests/Common/TestUtility.cs

```diff
@@ -0,0 +1,155 @@
+// Copyright 2023 Unity Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using NUnit.Framework;
+using pxr;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using USD.NET;
+using USD.NET.Unity;
+using USDScene = USD.NET.Scene;
+
+namespace Unity.Formats.USD
+{
+    public static class TestUtility
+    {
+#if UNITY_EDITOR
+        public static USDScene OpenUSDSceneWithGUID(string usdGUID, UsdStage.InitialLoadSet loadSet = UsdStage.InitialLoadSet.LoadNone)
+        {
+            var usdPath = Path.GetFullPath(AssetDatabase.GUIDToAssetPath(usdGUID));
+            return OpenUSDSceneWithFullPath(usdPath, loadSet);
+        }
+
+        // TODO: If materialImportMode = MaterialImportMode.ImportPreviewSurface, it creates all the texture2d files on the root assets
```

Comment:
Maybe we can move this TODO to a ticket.

Reply:
added JIRA ticket ID to the TODO